server/controllers/bookmarkController.js

```diff
@@ -0,0 +1,84 @@
+import models from '../db/models';
+import utils from '../helpers/Utilities';
+
+/**
+ * @Module BookmarkController
+ * @description Controlls all activities related to bookmarks
+ */
+class BookmarkController {
+  /**
+   * @description Bookmarks an article for reading later
+   * @param {Object} req - request object
+   * @param {Object} res - response object
+   * @returns {Object} Success Message
+   */
+  static async bookmarkArticle(req, res) {
+    const userId = req.user.id;
+    const { articleId } = req.params;
+    const article = await models.Article.findByPk(articleId);
+    if (!article) return utils.errorStat(res, 404, 'Article not found');
+    const bookmarked = await models.Bookmark.findOne({ where: { articleId } });
+    if (bookmarked) return utils.errorStat(res, 409, 'Aticle already bookmarked');
+    const addBookmark = await models.Bookmark.findOrCreate({
+      where: { articleId, userId },
+      defaults: { articleId, userId }
+    });
+    return utils.successStat(res, 201, 'Article bookmarked successfully', addBookmark);
```

Comment:
Can you change the property name for the response @obayomi96

Reply:
Done that now.